README.md

```diff
@@ -1,2 +1,321 @@
-# Quantum Network Explorer CLI
-Command Line Interface to interact with the Quantum Network Explorer
+# Quantum Network Explorer ADK
+Application Development Kit to interact with the Quantum Network Explorer. With the QNE-ADK you can create your application and experiment and run it locally using the commands. 
```

Comment:
Actually there is no interaction yet. We should be clear about that.
Also 'you can create your application and experiment' should be (much) more explaining.
"run it locally using the commands" is not clear.

Try to read this as a newcomer. This is the first document for an application developer.